server/credentials.go

```diff
@@ -87,6 +87,13 @@ func getExpectedPermissions() []expectedPermission {
 				Type: "Role",
 			},
 		},
+		{
```

Comment:
I am guessing this means the plugin will break until this permission is added? Unless the admin gave extra permissions?

Reply:
Good question: no, it won't "break", but it also won't behave to spec. Without the permission:
* The logs will log errors daily
* The presence semantics won't apply, and we'll always notify users (also clearly noted in logs)